api/external/compliance/reporting/reporting.proto

```diff
@@ -1021,7 +1065,26 @@ message Assets {
 	//Last run details of asset
 	string last_run = 4;
 }
+
 message AssetListResponse {
 	//Assets list as per the filters applied
 	repeated Assets assets = 1;
 }
+
+// ComplianceConfigRequest to update no of days in the config
+message ComplianceConfigRequest{
+	//No of days for the config 
+    int32 no_of_days = 1;
```

Comment:
We can change this to policy_value, as a user prospective no_of_days doesn't sound valid.